tests/zfs-tests/tests/functional/cli_root/zfs/zfs_003_neg.ksh

```diff
@@ -40,12 +40,12 @@
 
 verify_runnable "global"
 
-log_assert "zfs fails with unexpected scenarios."
+log_assert "zfs fails with unexpected scenario."
 
 #verify zfs failed if ZFS_DEV cannot be opened
 ZFS_DEV=/dev/zfs
 
-for file in $ZFS_DEV $MNTTAB; do
+for file in $ZFS_DEV; do
```

Comment:
The `is_linux` conditionals from https://github.com/rlaager/zfs/commit/79fd00afa25a72711972c20af8b2e96f5a1932d0 are missing.